test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,23 @@
+const { expect } = require('chai');
+const axios = require('axios');
+
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+describe('idempotence', () => {
+  it('PUT ', async () => {
+    const response = await object.put('https://api.github.com/user/following/aperdomob');
+    expect(response.status).to.equal(204);
+    expect(response.data).to.equal('');
+  });
+  it('Following test', async () => {
+    const response = await object.get('https://api.github.com/user/following');
+    expect(response.data.find((user) => user.login === 'aperdomob')).to.not.eql(undefined);
+  });
+  it('refollowing test', async () => {
+    const response = await object.get('https://api.github.com/user/following');
```

Comment:
please also call the put, the idea to test the idempotence is that, if we perform the same action several times (in this case the call to the put method), the result always will be the same (in this case that we continue following the desired user. So, the steps needed in this `it` section are:

1. Call again the put method (missing)
2. Call again the get method (done)
3. Verify you continue following the user (done)

Reply:
redone